### ISSUE TITLE
Pause between selectors

### ISSUE THREAD

**Issue by davthu** [garris/BackstopJS#738]
I have a bunch of components on a single page that I capture using a the selectors option. However, since the components are lazy loaded they don't show up in the report. Is it possible to add a waitFor option in between each element in a selector?

**Comment by garris** [comment]
Yes.  this is totally possible.   There are many many many ways to do this -- here's an example of an onReady script you could add using the puppeteer engine:

```
module.exports = async (page, scenario) => {
    await page.waitFor(<your first selector>);
    await page.waitFor(<your second selector>);
    await page.waitFor(<your nth selector etc...>);
};
```

You can check this directory for other example files depending on your engine https://github.com/garris/BackstopJS/tree/master/test/configs/backstop_data/engine_scripts
